pom.xml

```diff
@@ -30,8 +30,8 @@
 
   <dependencies>
     <dependency>
-      <groupId>org.spigotmc</groupId>
-      <artifactId>spigot</artifactId>
+      <groupId>com.destroystokyo.paper</groupId>
+      <artifactId>paper</artifactId>
```

Comment:
Does this make paper a hard requirement for building? paper-api is already inherited from style-guide/parent, no?

Reply:
It would, and no, neither paper nor spigot were added as dependencies within the parent pom, though the repositories were added for easy dependency addition.

Reply:
> though the repositories were added for easy dependency addition.

Ah yeah right

> It would

I don't think that's desireable then. It makes Paper a hard requirement for the entire civ stack.

Reply:
Without it, we wont be able to use Paper's de-obfuscation methods or any custom features

Reply:
Do we _need_ any of  those? As far as this PR goes it makes things a bit nicer and NMS a bit rarer, but that's it

Reply:
It *is* more of a want than an explicit need, but as you said the other day, there's no reason not to use it by now :P